pages/1.12/cli/command-reference/dcos/index.md

```diff
@@ -20,22 +22,29 @@ dcos [options] [<command>] [<args>...]
 
 # Options
 
-*Table 1 - Options*
-
 | Name | Description |
 |---------|-------------|-------------|
-| `--debug`   |  Enable debug mode. |
 | `--help, h`   |  Display usage. |
-| `--log-level=<log-level>`  | Set the logging level. This setting does not affect the output sent to `stdout`.  |
-|  `--version, v`  |  Display version information.  |
-| `<log-level>` | The severity levels are shown in Table 2. |
+|  `--debug`  |  Enable debug mode. |
+| `--version` | Display version information. |
+| `--log-level=<log-level>`  | Set the logging level. This setting does not affect the output sent to `stdout`.  See below for severity levels. |
+
+
+## Log level severity
```

Comment:
This seems outdated. The 1.12 CLI uses `-v` and `-vv` for verbosity of logging. The `--log-level` flag still works but was deprecated with 1.12.